Source/WebCore/dom/Element.h

```diff
@@ -800,7 +800,7 @@ class Element : public ContainerNode {
     bool childTypeAllowed(NodeType) const final;
 
     void notifyAttributeChanged(const QualifiedName&, const AtomString& oldValue, const AtomString& newValue, AttributeModificationReason = AttributeModificationReason::Directly);
-    enum SynchronizationOfLazyAttribute { NotInSynchronizationOfLazyAttribute, InSynchronizationOfLazyAttribute };
+    enum class SynchronizationOfLazyAttribute : bool { No, Yes };
```

Comment:
SynchronizeLazyAttribute might be shorter, but no strong preference.

Reply:
Oh, this flag doesn't mean to trigger a synchronization. It's communicating whether we're in the midst of synchronization or not.

Reply:
I would have called it `InSynchronizationOfLazyAttribute : bool { No, Yes }`. The newly proposed name is a bit confusing.